tests/components/satel_integra/conftest.py

```diff
@@ -58,12 +60,18 @@ def mock_config_entry() -> MockConfigEntry:
         title="192.168.0.2",
         data=MOCK_CONFIG_DATA,
         options=MOCK_CONFIG_OPTIONS,
-        entry_id="SATEL_INTEGRA_CONFIG_ENTRY_1",
+        entry_id=MOCK_ENTRY_ID,
         version=1,
         minor_version=1,
     )
 
 
+@pytest.fixture
+def entity_reg(hass: HomeAssistant):
+    """Return an empty, loaded, registry."""
+    return mock_registry(hass)
```

Comment:
I think @joostlek was referring to the standard "entity_registry" fixture already available for all components.